R/run_mcmc.R

```diff
@@ -110,14 +110,26 @@ run_mcmc_p <- function(dt, priorObj, n.chains, n.adapt, n.burn, n.iter, seed, pa
     cat("seed_list:", seed_list, "number of dataset", length(dt), "\n")
   )
 
-  message(n.cores, " clusters are being used")
+  ps_message(n.cores, " clusters are being used")
   cl <- parallel::makeCluster(n.cores)
   doParallel::registerDoParallel(cl)
 
+  # Load psborrow on the clusters
+  local.psborrow.quiet <- getOption('psborrow.quiet') # To pass to clusters
+  parallel::clusterExport(cl, varlist =  c('local.psborrow.quiet'), envir = environment())
+  parallel::clusterEvalQ(cl, {
+    try( # For development purposes, we will have to call `devtools::load_all()`
+      if(grep('psborrow',rstudioapi::getActiveProject())) {
```

Comment:
You can probabally replace the use of `rstudioapi::getActiveProject()` with something like this (which is just base R)

```
is_psborrow_dev <- function() {
    desc <- file.path("DESCRIPTION")
    if (file.exists(desc)) {
        content <- read.dcf(desc)
        pkg <- content[1, ][["Package"]]
        return(pkg == "psborrow")
    }
    return(FALSE)
}
```